docs/kyma/04-09-upgrade.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Upgrade Kyma
+type: Installation
+---
+
+Upgrading Kyma is the process of migrating from one version of the software to a newer release. This operation depends on [release artifacts](https://github.com/kyma-project/kyma/releases) listed in the "assets" section of the GitHub releases page and migration guides delivered with the target release.
+
+To upgrade to a version that is several releases newer than the version you're currently using, you must move up to the desired release incrementally. You can skip patch releases.
+
+For example, if you're running Kyma 1.0 and you want to upgrade to version 1.3, you must perform these operations:
+  1. Upgrade from version 1.0 to version 1.1.
+  2. Upgrade from version 1.1 to version 1.2.
+  3. Upgrade from version 1.2 to version 1.3.  
+
+>**NOTE:** Kyma does not support a dedicated downgrade procedure. You can achieve a similar result by restoring your cluster from a backup. Read [this](/components/backup/#overview-overview) document to learn more about backups.
+
+The upgrade procedure relies heavily on Helm. As a result, the availability of cluster services during the upgrade is not defined by Kyma and can vary from version to version. The existing Custom Resources (CRs) remain in the cluster.
+
+>**NOTE:** To learn more about the technical aspects of the upgrade, read [this](https://github.com/kyma-project/kyma/blob/master/components/kyma-operator/README.md) document.
+
+## Upgrade your deployment to a newer version
```

Comment:
Is upgrading deployment and upgrading Kyma the same thing? Can we say 
```
## Upgrade your Kyma deployment to a newer version
```
or 
```
## Upgrade Kyma to a newer version
```
?

Reply:
```suggestion
## Upgrade Kyma to a newer version
```